src/components/ItemEditForm/EditItemModal.jsx

```diff
@@ -0,0 +1,206 @@
+import { Dialog, Transition } from '@headlessui/react';
+import { Fragment, useState } from 'react';
+import { useForm, Controller } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+import { TbEdit } from 'react-icons/tb';
+import { Input, TextArea, SubmitButton, ListBox, ComboBox } from '../Forms';
+
+const schema = yup.object().shape({
+  name: yup.string().required('Cannot stay empty.'),
+  desc: yup.string().required('Cannot stay empty.'),
+});
+
+const ITEM_TYPES = ['Crafted', 'Donated', 'New', 'Used'];
+
+const ITEM_CATEGORY = [
+  'Men',
+  'Women',
+  'Kids',
+  'Toys',
+  'Books',
+  'Sports',
+  'Clothes',
+  'Furniture',
+  'Electronics',
+  'Homemade',
+  'Other',
+];
```

Comment:
This is iterating in both the new modal and our currently merged form, can you move these inside a "constants" file in src/utils in both components, so DRY code.